### PR TITLE
php72-phalcon 3.2.0 (new formula)

### PR DESCRIPTION
**Pull request by romainneutron**
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-php/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

**Comment by richmilns** [comment]
I noticed that Phalcon have recently released 3.2.0 - any chance this could be updated in your PR?

File: Formula/php72-phalcon.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+require File.expand_path("../../Abstract/abstract-php-extension", __FILE__)
+
+class Php72Phalcon < AbstractPhp72Extension
+  init
+  desc "Full-stack PHP framework"
+  homepage "https://phalconphp.com/"
+  url "https://github.com/phalcon/cphalcon/archive/v3.1.2.tar.gz"
```
Comment (javian) [inline Formula/php72-phalcon.rb]: Please update to latest version

File: Formula/php72-phalcon.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  sha256 "da619cab5979b3592b5e880a5e286e8d59841e911000be944fcddb030a7af587"
+  head "https://github.com/phalcon/cphalcon.git"
+
+  depends_on "pcre"
+
+  def install
+    if MacOS.prefer_64_bit?
```
Comment (javian) [inline Formula/php72-phalcon.rb]: Please remove this conditional statement to assume 64bit, 32bit is no longer required.

**Comment by romainneutron** [comment]
PR updated, comments addressed

**Comment by javian** [comment]
@romainneutron I think you took away too much code. See https://github.com/Homebrew/homebrew-php/blob/master/Formula/php70-phalcon.rb

**Comment by javian** [comment]
Thanks @romainneutron !

**Pull request closed without merging**